### PR TITLE
11 rubocop

### LINKED ISSUE #11: Rubocop
Configure rubocop to check build.

### PR DESCRIPTION
**Pull request by AlexNimoy**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -7,3 +7,6 @@ Style/FrozenStringLiteralComment:
 
 Style/Documentation:
   Enabled: false
+
+Metrics/MethodLength:
+  Max: 20
```
Comment (sergio-fry) [inline .rubocop.yml]: Do we have such long methods already? Lets try to keep it short for now

**Review by sergio-fry — CHANGES_REQUESTED** [review]
Looks great, minor comment

File: .github/workflows/rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Run RuboCop
+
+on: [push, pull_request]
```
Comment (sergio-fry) [inline .github/workflows/rubocop.yml]: Look like we have rubocoped executed twice. Why do we need it?

File: .rubocop.yml (hunk 1 above)
Comment (AlexNimoy) [reply .rubocop.yml]: Yes we have.
lib/bddsm/progress_report.rb:18:5: C: Metrics/MethodLength: Method has too many lines. [11/10]

Comment (sergio-fry) [reply .rubocop.yml]: lets add a todo, it would be great to fix it later

**Review by sergio-fry — APPROVED** [review]

**Pull request merged**